xml_inscriptions/transcribed/MA.Bos.MFA.G.01.Res.08.34b.xml

```diff
@@ -113,49 +106,51 @@
                   <biblScope unit="item">no. 231</biblScope>
                </bibl>
                <bibl>
-                  <ref type="external" target="xx">Title of Reference</ref>
+                  <ref type="external"
+                     target="https://scholarlyeditions-brill-com.revproxy.brown.edu/reader/urn:cts:greekDoc:seg.028-0231.sego:lem-ap/"
+                     >Supplementum Epigraphicum Graecum web entry</ref>
                </bibl>
             </listBibl>
          </sourceDesc>
       </fileDesc>
       <encodingDesc>
          <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
-                      href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
+            href="http://library.brown.edu/usep_data/resources/include_taxonomies.xml">
             <xi:fallback>
                <p>Taxonomies for US Epigraphy controlled values</p>
             </xi:fallback>
          </xi:include>
       </encodingDesc>
       <profileDesc/>
       <revisionDesc>
-         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009 EpiDoc</change>
+         <change when="2010-09-29" who="Center for Digital Scholarship">Converted to September 2009
+            EpiDoc</change>
          <change when="2011-05-13" who="Elli Mylonas">More tweaks for bib only inscriptions</change>
          <change when="2023-06-23" who="Jean Wanlass">adding information to bib only stub</change>
       </revisionDesc>
    </teiHeader>
    <!--  *************************************  <facsimile>  *********************************  -->
    <!-- Used to record images of an inscription.   -->
-   
    <facsimile>
       <surface>
-         <desc>photo: MFA Res.08.34b</desc>
+         <desc/>
          <graphic url="MA.Bos.MFA.G.Res.08.34b.jpg"/>
       </surface>
    </facsimile>
-   
    <text>
       <body>
          <div type="edition">
-            <ab><supplied reason="lost">ΔI</supplied>EIYΨICTШΔAΦNIC <supplied reason="lost">EU</supplied>XHN ANEϴHKE</ab></div>
-         
+            <ab><supplied reason="lost">Δι</supplied>εὶ ὑψίστω Δάφνις <supplied reason="lost">εὐ</supplied>χὴν ἀνέθηκε</ab>
+         </div>
          <div type="translation">
-            <p>to Zeus Hypsistos Daphnis dedicated [this in fulfillment of] a vow</p></div>
-         
+            <p>to <persName role="god">Zeus</persName> <persName>Hypsistos Daphnis</persName> dedicated [this in fulfillment of] a vow</p>
```

Comment:
Hypsistos should go with Zeus, not Daphnis; make sure it's within the correct `<persName>`